file_reader.go

```diff
@@ -184,6 +209,25 @@ func (f *FileReader) Read(b []byte) (int, error) {
 		return 0, io.ErrClosedPipe
 	}
 
+	offset := f.offset
+	n, err := f.readImpl(b)
```

Comment:
This reads the encrypted bytes into b. If the unencrypted text is smaller than the encrypted text, we'll have leftover garbage in `b`. We should (re)use a `bytes.Buffer` here if doing decryption.

Reply:
this must not happen: encrypted text size for `ctr` mode must be the same as unencrypted.